x-pack/legacy/plugins/ml/public/data_visualizer/components/field_data_card/metric_distribution_chart/metric_distribution_chart.tsx

```diff
@@ -0,0 +1,147 @@
+
+export interface MetricDistributionChartData {
+  x: number;
+  y: number;
+  dataMin: number;
+  dataMax: number;
+  percent: number;
+}
+
+interface Props {
+  width: number;
+  height: number;
+  chartData: MetricDistributionChartData[];
+  fieldFormat?: any; // Kibana formatter for field being viewed
+}
+
+const SPEC_ID = 'metric_distribution';
+
+const IS_DARK_THEME = chrome.getUiSettingsClient().get('theme:darkMode');
+const themeName = IS_DARK_THEME ? darkTheme : lightTheme;
+const EVENT_RATE_COLOR = themeName.euiColorVis1;
+const seriesColorValues: DataSeriesColorsValues = {
+  colorValues: [],
+  specId: getSpecId(SPEC_ID),
+};
+const seriesColors = new Map([[seriesColorValues, EVENT_RATE_COLOR]]);
+
+export const MetricDistributionChart: FC<Props> = ({ width, height, chartData, fieldFormat }) => {
+  // This value is shown to label the y axis values in the tooltip.
+  // Ideally we wouldn't show these values at all in the tooltip,
+  // but this is not yet possible with Elastic charts.
+  const seriesName = i18n.translate('xpack.ml.fieldDataCard.metricDistributionChart.seriesName', {
+    defaultMessage: 'distribution',
+  });
+
+  const headerFormatter: TooltipValueFormatter = (tooltipData: TooltipValue) => {
+    const xValue = tooltipData.value;
+    const chartPoint: MetricDistributionChartData | undefined = chartData.find(
+      data => data.x === xValue
+    );
+
+    return (
+      <MetricDistributionChartTooltipHeader
+        chartPoint={chartPoint}
+        maxWidth={width / 2}
+        fieldFormat={fieldFormat}
+      />
+    );
+  };
+
+  const tooltipProps = {
+    headerFormatter,
+  };
+
+  return (
+    <div style={{ width, height }}>
+      <Chart>
+        <Settings
+          theme={{
+            areaSeriesStyle: {
+              line: {
+                stroke: 'red',
+                strokeWidth: 1,
+                visible: true,
+              },
+              point: {
+                visible: false,
+                radius: 0,
+                opacity: 0,
+              },
+              area: { fill: 'red', visible: true, opacity: 1 },
+            },
+          }}
+          tooltip={tooltipProps}
+        />
+        <Axis
+          id={getAxisId('bottom')}
+          position={Position.Bottom}
+          tickFormat={d => kibanaFieldFormat(d, fieldFormat)}
+        />
+        <Axis
+          id={getAxisId('left')}
+          position={Position.Left}
+          tickFormat={d => d.toFixed(3)}
+          hide={true}
+        />
+        <AreaSeries
+          id={getSpecId(SPEC_ID)}
+          name={seriesName}
+          xScaleType={ScaleType.Linear}
+          yScaleType={ScaleType.Linear}
+          xAccessor="x"
+          yAccessors={['y']}
+          data={chartData.length > 0 ? chartData : [{ x: 0, y: 0 }]}
+          curve={CurveType.CURVE_STEP_AFTER}
+          // TODO - switch to use inline areaSeriesStyle to set series fill once charts version is 8.0.0+
```

Comment:
Assuming these TODOs are going to be done in a follow up?

Reply:
Yes, using the inline styling of the series color in here and the document count chart will be done in a follow-up. It is awaiting the version of Elastic charts to be bumped to 8.0.0+